App Crashing when DataSensorLogger is tried to opened

**Actual Behaviour*

The app crashes when Data Sensor Logger is tried to open. giving a runtime error.

**Expected Behaviour**

The app should not crash.

**Steps to reproduce it**

Navigation Menu -> Applications -> Data Sensor Logger

**LogCat for the issue**

java.lang.RuntimeException: Unable to start activity ComponentInfo{org.fossasia.pslab/org.fossasia.pslab.activity.SensorDataLoggerActivity}: java.lang.IllegalStateException: Required view 'snackbar_text' with ID 2131689702 for field 'snackbarTextView' was not found. If this view is optional add '@Nullable' (fields) or '@Optional' (methods) annotation.
                                                                        at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2724)
                                                                        at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2785)
                                                                        at android.app.ActivityThread.-wrap12(ActivityThread.java)
                                                                        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1532)
                                                                        at android.os.Handler.dispatchMessage(Handler.java:102)
                                                                        at android.os.Looper.loop(Looper.java:163)
                                                                        at android.app.ActivityThread.main(ActivityThread.java:6342)
                                                                        at java.lang.reflect.Method.invoke(Native Method)
                                                                        at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:880)
                                                                        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:770)
                                                                     Caused by: java.lang.IllegalStateException: Required view 'snackbar_text' with ID 2131689702 for field 'snackbarTextView' was not found. If this view is optional add '@Nullable' (fields) or '@Optional' (methods) annotation.
                                                                        at butterknife.internal.Utils.findRequiredView(Utils.java:92)
                                                                        at butterknife.internal.Utils.findRequiredViewAsType(Utils.java:104)
                                                                        at org.fossasia.pslab.activity.SensorDataLoggerActivity_ViewBinding.<init>(SensorDataLoggerActivity_ViewBinding.java:34)
                                                                        at java.lang.reflect.Constructor.newInstance0(Native Method)
                                                                        at java.lang.reflect.Constructor.newInstance(Constructor.java:430)
                                                                        at butterknife.ButterKnife.createBinding(ButterKnife.java:199)
                                                                        at butterknife.ButterKnife.bind(ButterKnife.java:124)
                                                                        at org.fossasia.pslab.activity.SensorDataLoggerActivity.onCreate(SensorDataLoggerActivity.java:90)
                                                                        at android.app.Activity.performCreate(Activity.java:6847)
                                                                        at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1119)
                                                                        at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2677)
                                                                        at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2785) 
                                                                        at android.app.ActivityThread.-wrap12(ActivityThread.java) 
                                                                        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1532) 
                                                                        at android.os.Handler.dispatchMessage(Handler.java:102) 
                                                                        at android.os.Looper.loop(Looper.java:163) 
                                                                        at android.app.ActivityThread.main(ActivityThread.java:6342) 
                                                                        at java.lang.reflect.Method.invoke(Native Method) 
                                                                        at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:880) 
                                                                        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:770) 


**Screenshots of the issue**

Not applicable

**Would you like to work on the issue?**

Yes.

--- a/app/src/main/java/org/fossasia/pslab/activity/SensorDataLoggerActivity.java
+++ b/app/src/main/java/org/fossasia/pslab/activity/SensorDataLoggerActivity.java
@@ -79,7 +79,7 @@ public class SensorDataLoggerActivity extends AppCompatActivity {
     CoordinatorLayout coordinatorLayout;
     @BindView(R.id.layout_container)
     FrameLayout container;
-    @BindView(android.support.design.R.id.snackbar_text)
+    @Nullable @BindView(android.support.design.R.id.snackbar_text)
     TextView snackbarTextView;
 
 

